src/GCHP_GridComp/GCHPctmEnv_GridComp/GCHPctmEnv_GridCompMod.F90

```diff
@@ -660,53 +634,109 @@ subroutine Run ( GC, IMPORT, EXPORT, CLOCK, RC )
       End Do
       End Do
 
-
-      ! Arrays were calculated so that 1 = Surface
-      ! FV3 wants 1 = TOA, LM+1 = Surface
-      ! Vertically flip all the arrays to accomplish this      
+      ! Flip vertically so that exports have top-down indexing (as per AdvCore_GridComp imports)      
       DryPLE0r8(:,:,:) = DryPLE0r8(:,:,LM:0:-1)
       DryPLE1r8(:,:,:) = DryPLE1r8(:,:,LM:0:-1)
       PLE0r8   (:,:,:) = PLE0r8   (:,:,LM:0:-1)
       PLE1r8   (:,:,:) = PLE1r8   (:,:,LM:0:-1)
-      UC       (:,:,:) =  UC      (:,:,LM:1:-1)
-      VC       (:,:,:) =  VC      (:,:,LM:1:-1)
-
-      DEALLOCATE( AP, BP )
+      SPHU0r8  (:,:,:) = 1.0d0*SPHU0(:,:,LM:1:-1)
```

Comment:
What is the reasoning behind multiplying by 1 to cast rather than use DBLE?  Also, SPHU0 is SPHU0_IMPORT vertically flipped, and then here SPHU0 is vertically flipped for SPHU0r8. Could this be consolidated to avoid twice?

Reply:
Multiplying `1.0d` was the nearby convention so I used that. I'll switch them to `dble` casts.

There is a double-flip in the case of top-down meteorology. I thought the logic was clearer this way, so that metfield orientation, and the export re-orienting were handled in separately.

Would you still favor the consolidation?

Reply:
I am still in favor of consolidation because (1) the flip during casting is sort of hidden, and (2) flipping has a performance cost and while minor I think every ding counts. In general I suggest going heavy on the commenting of all the flipping steps to explain the logic.

Reply:
Added in #181 (merged into this PR)